tools/Az.Tools.Predictor/Az.Tools.Predictor/Settings.cs

```diff
@@ -31,16 +31,30 @@ sealed class Settings
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
         };
 
+        /// <summary>
+        /// The maximum number of suggestions that have the same command name.
+        /// </summary>
+        public int? MaxAllowedCommandDuplicate { get; set; }
```

Comment:
Would it make more sense to have these default to something if not set instead of being null?

Reply:
It's not being null. The default values are set in the file AzPredictorSettings.json.
I'll create a task to set the default values in the code itself instead of reading from the file.